packages/auth/amplify_auth_cognito_dart/lib/src/state/machines/sign_in_state_machine.dart

```diff
@@ -70,7 +70,7 @@ class SignInStateMachine extends StateMachine<SignInEvent, SignInState> {
   bool get hasIdentityPool => identityPoolConfig != null;
 
   /// The Cognito Identity Provider service client.
-  late final CognitoIdentityProviderClient cognitoIdentityProvider = expect();
+  late final CognitoIdentityProviderClient _cognito = expect();
```

Comment:
"Cognito" is not descriptive enough since we work with two Cognito APIs, one for user pools (called `CognitoIdentityProvider`) and one for identity pools (called `CognitoIdentity`)

Reply:
Ok, shouldn't it be private though?